README.md

```diff
@@ -61,7 +61,7 @@ To update the types from the Open API docs run the following commands:
 
 `npx openapi-typescript https://prisoner-offender-search-dev.prison.service.justice.gov.uk/v3/api-docs -o server/@types/prisonerSearchApi/index.d.ts`
 
-`npx openapi-typescript https://raw.githubusercontent.com/ministryofjustice/curious-API/main/curious-api-specification.yaml -o server/@types/curiousApi/index.d.ts`
+`npx openapi-typescript https://testservices.sequation.net/sequation-virtual-campus2-api/v3/api-docs -o server/@types/curiousApi/index.d.ts`
```

Comment:
Took the opportunity to use the official location of the curious openAPI spec; rather than the one I found in a MoJ repo !!!
(they are the same file though 👍 )